.ci.yaml

```diff
@@ -48,6 +48,26 @@ platform_properties:
         ]
       os: Linux
       device_os: "N"
+  linux_samsung:
```

Comment:
Should we call this something generic like Linux_android_high_end?

Samsung has both high end and low end phones, so for new contributors it may be confusing

Reply:
One way is append `s10` to make it more clear.
We may have different `high_end` phones in the future, like pixel. As for `low_end`, we may also have multiple, like `samsung A02`, existing `motog4`.

Reply:
s10 sounds good to me

Reply:
Agree with @CaseyHillers.

Reply:
Cool. Appended.